webapp/sources/ldap-inventory/inventory-api/src/main/scala/com/normation/inventory/services/provisioning/InventoryParser.scala

```diff
@@ -76,6 +79,35 @@ trait InventoryParser {
 
 }
 
+/*
+ * Use a safe SAX parser
+ */
+object XmlInventoryParser {
+
+  /*
+   * Create a safe sax parser using https://cheatsheetseries.owasp.org/cheatsheets/XML_External_Entity_Prevention_Cheat_Sheet.html#jaxp-documentbuilderfactory-saxparserfactory-and-dom4j
+   * guiding lines.
+   */
+  val saxParserFactory: SAXParserFactory = {
+    try {
+      val f = SAXParserFactory.newInstance()
+      f.setFeature(XMLConstants.FEATURE_SECURE_PROCESSING, true)
+      f.setNamespaceAware(false)
```

Comment:
this one is from the default implementation in Scala, I'm keeping it because I don't want to try to understand what would be the side effects of changing it.